mapper/tedge_mapper/src/main.rs

```diff
@@ -18,27 +39,82 @@ async fn main() -> Result<(), anyhow::Error> {
         .with_span_events(tracing_subscriber::fmt::format::FmtSpan::CLOSE)
         .init();
 
-    info!("{} starting!", APP_NAME);
+    match cloud_name {
+        "c8y" => {
+            let _flockfile = check_another_instance_is_running(APP_NAME_C8Y)?;
 
-    let _lockfile = match flockfile::Flockfile::new_lock(format!("{}.lock", APP_NAME)) {
-        Ok(file) => file,
-        Err(err) => {
-            error!("Another instance of {} is running.", APP_NAME);
-            return Err(err.into());
+            info!("{} starting!", APP_NAME_C8Y);
+
+            let mqtt_config = mqtt_client::Config::default();
+            let mqtt = Client::connect(APP_NAME_C8Y, &mqtt_config).await?;
```

Comment:
One can improve code sharing here by adding an `app_name` property to the `MapperConfig` struct.

Reply:
The `instrument` should be happy with any `&str`.

Reply:
It should be a constant value.
```
error[E0435]: attempt to use a non-constant value in a constant
  --> mapper/tedge_mapper/src/main.rs:65:37
   |
65 |             .instrument(debug_span!(aaa.as_str()))
   |                                     ^^^ non-constant value
```